server/views/partials/prisonerBanner.njk

```diff
@@ -1,17 +1,23 @@
-<div class="app-prisoner-banner govuk-!-display-none-print">
```

Comment:
Minor, do we need to get rid of the `app-prisoner-banner` styles? (assuming they were custom styles we wrote, rather than ones from GDS ?)

Reply:
There were no styles for `app-prisoner-banner` in our service (would have been a custom class)